whisper/whisper-v3-truss/model/model.py

```diff
@@ -14,23 +16,46 @@ def __init__(self, **kwargs):
         self.device = "cuda" if torch.cuda.is_available() else "cpu"
         self.model = None
 
-    def preprocess(self, request: Dict) -> Dict:
-        resp = requests.get(request["url"])
-        return {"response": resp.content}
+        def audio_url_to_waveform(self, path_or_url: str):
```

Comment:
i think the indentation here is one level off

Reply:
good catch!